src/jllvm/vm/VirtualMachine.hpp

```diff
@@ -142,6 +140,9 @@ class VirtualMachine
         return invokeJava<Ret>(addr, args...);
     }
 
+    /// Performs class initialization for 'classObject'. This is a noop if 'classObject' is not uninitialized.
```

Comment:
```suggestion
    /// Performs class initialization for 'classObject'. This is a noop if 'classObject' is already initialized.
```

Reply:
The current sentence is more accurate I believe since it is also a noop if the class is currently under initialization rather than just uninitialized :)